srcpkgs/alacritty/template

```diff
@@ -1,18 +1,16 @@
 # Template file for 'alacritty'
 pkgname=alacritty
-version=0.1.0.20180902
+version=0.2.0
 revision=1
-_commit=865727c062810e29fa33b5c04bb05510e7da3ddf
-wrksrc="${pkgname}-${_commit}"
 hostmakedepends="cargo cmake pkg-config"
 makedepends="freetype-devel fontconfig-devel"
 depends="libXxf86vm xclip ncurses"
 short_desc="Cross-platform, GPU-accelerated terminal emulator"
 maintainer="Andrea Brancaleoni <abc@pompel.me>"
 license="Apache-2.0"
-homepage="https://github.com/jwilm/alacritty"
-distfiles="${homepage}/archive/${_commit}.tar.gz"
-checksum=30c7f4ae6964e8009f5b6e2741f7df25f0e527f7be9f9a1751af8e6ba1c08d26
+homepage="https://github.com/jwilm/${pkgname}"
```

Comment:
Alacritty now keeps a changelog https://raw.githubusercontent.com/jwilm/alacritty/master/CHANGELOG.md

Reply:
Alright. Although, shouldn't that be:

```
changelog=https://raw.githubusercontent.com/jwilm/alacritty/v${version}/CHANGELOG.md
```

Reply:
that won't work since github uses the git hash not git tag in the url
the changelog on master should contain all previous releases so it always has the same or more info

Also, seems like i686 is broken due to an upstream library? (not sure, I've never used rust)

if that's the case, add something like 
```
case "${XBPS_TARGET_MACHINE}" in
	i686)
		broken="winit library broken on i686
		See https://travis-ci.org/void-linux/void-packages/jobs/429814888";
		;;
esac
```